Makefile

```diff
@@ -24,11 +26,21 @@ test-%:
 docker-build: $(patsubst %, docker-build-%, $(DOCKERFILES))
 
 docker-build-%:
-	docker build -t quay.io/kubevirt/device-plugin-$*:latest cmd/$(subst -,/,$*)
+	@cp ${GOPATH}/bin/${notdir $(subst -,/,$*)} ./cmd/$(subst -,/,$*)
+	docker build -t ${REGISTRY}/device-plugin-$*:latest ./cmd/$(subst -,/,$*)
 
 docker-push: $(patsubst %, docker-push-%, $(DOCKERFILES))
 
 docker-push-%:
-	docker push quay.io/kubevirt/device-plugin-$*:latest
+	docker tag ${REGISTRY}/device-plugin-$*:latest ${REGISTRY}/device-plugin-$*
```

Comment:
is it needed to tag it? is not it tagged by the name automatically?

Reply:
not sure... will check

Reply:
you are right. tag is not needed